scripts/should_rebuild_master.py

```diff
@@ -21,12 +21,8 @@ def should_rebuild_project(changed_files, project):
     if should_retest_project(changed_files, project):
         return True
 
-    elif ('Dockerfile' in changed_files) or ('.dockerignore' in changed_files):
```

Comment:
1. Did you mean to throw out the `.dockerignore`?
2. This isn't valid Python syntax (unmatched left parens), so this script will always exit with code 1 and a SyntaxError – which defaults to always rebuilding.